### DISCUSSION TITLE
Programming the LGT8F328P with Raspberry Pi

### DISCUSSION THREAD

**Discussion by just-jason** [dbuezas/lgt8fx#211] [Q&A]
There are some excellent guides for using another LGT8F board as an ISP programmer, but I was looking at the possibility of using the Pi as a programmer along the lines of this guide 
[https://learn.adafruit.com/program-an-avr-or-arduino-using-raspberry-pi-gpio-pins](url)  
I have  tested this with the Pi > LGT8F328p connected as follows
| Pi  | LGT8F  |
| :-: | :-: |
| RST | RST |
| SCK | SWC |
| MISO | SWD |

but was not able to get it to recognize the board, even when running the basic test command to read the fuses :
`sudo avrdude -p atmega328p -C ~/avrdude_gpio.conf -c pi_1 -v` 

Does anyone know if the basic avrdude ISP flashing process for a atmega328p can be used for the LGT328p, or is there a difference that I need to be aware of ?

**Comment by dbuezas** [comment]
Yes ISP is different.
see here: https://github.com/dbuezas/lgt8fx/blob/master/lgt8f/libraries/LarduinoISP/readme.md

I guess you could port the sourcecode of the isp sketch [here](https://github.com/brother-yan/LGTISP/blob/master/LGTISP.ino) if you dare :)
